lib/api/controllers/util/requestAssertions.js

```diff
@@ -80,6 +80,15 @@ module.exports = {
   /**
    * @param {Request} request
    */
+  assertIdStartsNotUnderscore: function requestAssertHasId (request) {
```

Comment:
and rename `function requestAssertHasId ` (which is copied/pasted from the previous function) by `function requestAssertIdStartsNotUnderscore`